crates/modelardb_server/src/storage/mod.rs

```diff
@@ -432,6 +432,28 @@ impl StorageEngine {
             .await
     }
 
+    /// Mark the table with `table_name` as dropped in the data transfer component. This will prevent
+    /// data related to the table from being transferred to the remote data folder.
+    pub(super) async fn mark_table_as_dropped(&self, table_name: &str) {
```

Comment:
Maybe include model table in the name to make it clear it works for both.

Reply:
I have left it as it was since I prefer using `table` to refer to both normal tables and model tables and then be specific when we refer to either.

Reply:
I agree.